src/libraries/EventsLib.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {FlowConfig, SupplyConfig} from "../interfaces/IPublicAllocator.sol";
+
+/// @title EventsLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library exposing events.
+library EventsLib {
+    /// @notice Emitted when the public reallocation is triggered.
+    event PublicReallocate(address sender, uint256 fee);
```

Comment:
the fee cannot be something else actually
```suggestion
    event PublicReallocate(address sender);
```

Reply:
34cac9fb9e4994ae651490c4c09a8973e1051711